lib/lightning/accounts.ex

```diff
@@ -70,17 +71,26 @@ defmodule Lightning.Accounts do
   def perform(%Oban.Job{args: %{"type" => "purge_deleted"}}) do
     users_with_activities =
       from(run in Lightning.Run,
-        join: user in Lightning.Accounts.User,
+        join: user in User,
         on: run.created_by_id == user.id,
         select: user.id,
         distinct: true
       )
 
+    users_with_files =
+      from(file in File,
```

Comment:
```suggestion
      from(file in Projects.File,
```
Can we use `Projects.File` to avoid conflicts with the `File` module?